banner-plugin/src/admin/routes/banners/page.tsx

```diff
@@ -1,15 +1,173 @@
+                          </div>
+                          <Input {...field} placeholder="Enter banner title" />
+                        </div>
+                      )}
+                    />
+
+                    <Controller
+                      control={form.control}
+                      name="link"
+                      render={({ field }) => (
+                        <div className="flex flex-col space-y-2">
+                          <div className="flex items-center gap-x-1">
+                            <Label size="small" weight="plus">
+                              Link
+                            </Label>
+                          </div>
+                          <Input {...field} placeholder="Enter banner link" />
+                        </div>
+                      )}
+                    />
+
+                    <Controller
+                      control={form.control}
+                      name="image"
+                      render={({ field }) => (
+                        <div className="flex flex-col space-y-2">
+                          <div className="flex items-center gap-x-1">
+                            <Label size="small" weight="plus">
+                              Image
+                            </Label>
+                          </div>
+                          <Input {...field} placeholder="Enter image URL" />
+                        </div>
+                      )}
+                    />
+
+                    <Controller
+                      control={form.control}
+                      name="image_mobile"
+                      render={({ field: { value, onChange, ...field } }) => (
+                        <div className="flex flex-col space-y-2">
+                          <div className="flex items-center gap-x-1">
+                            <Label size="small" weight="plus">
+                              Mobile Image (Optional)
+                            </Label>
+                          </div>
+                          <Input {...field} value={value || ""} onChange={(e) => onChange(e.target.value || null)} placeholder="Enter mobile image URL" />
+                        </div>
+                      )}
+                    />
+
+                    <Controller
+                      control={form.control}
+                      name="sort_order"
+                      render={({ field: { value, onChange, ...field } }) => (
+                        <div className="flex flex-col space-y-2">
+                          <div className="flex items-center gap-x-1">
+                            <Label size="small" weight="plus">
+                              Sort Order
+                            </Label>
+                          </div>
+                          <Input {...field} type="number" value={value} onChange={(e) => onChange(Number(e.target.value))} placeholder="Enter sort order" />
+                        </div>
+                      )}
+                    />
```

Comment:
Could we have something similar to the "Edit ranking" in the categories? so we could have a sortable list (thus we could omit this field here)

<img width="1497" alt="Screenshot 2024-12-30 at 20 09 36" src="https://github.com/user-attachments/assets/9b6b2501-b05f-4b75-a522-2f2bfc29af89" />